hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/Route.java

```diff
@@ -272,9 +291,25 @@ private void init() {
     	getContext().setUuidGenerator(new TransactionIdGenerator());
     	injectProperties();
     	properties = ApplicationProperties.getInstance();
+		//isRTrans = isRTrans();
     	loadValidator();
     	
     }
+
+	/**
+	 * This method is used to append multiple Predicates for RTrans message type
+	 * Builds a compound predicate to use it in the Route
+	 */
+	private Predicate isRTrans() {
+		String r03 = properties.getValue(ApplicationProperty.RTRANS_R03_ENDPOINT);
+		String r07 = properties.getValue(ApplicationProperty.RTRANS_R07_ENDPOINT);
+		String r09 = properties.getValue(ApplicationProperty.RTRANS_R09_ENDPOINT);
```

Comment:
Maybe run this by Trevor but I don't see the benefit of making these properties. Having this level of indirection doesn't really provide any value as R03, R07, R09 are fixed values. And if you were to add a new RTrans type you would have to make updates to the code anyways. If they were truly "endpoints", i.e. the URL to which requests are sent then it would make sense but that is a separate property. If we want to dynamically support which messages types are RTrans (though I don't think this is a requirement) we could have a list of RTrans types in the properties (e.g. rtrans-message-types) which would allow the ability to change the types without making code changes.